addons_marketplace/marketplace-manager/marketplace_manager.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+from api.v1.marketplace_blueprint import marketplaceblp
+from db.mongodb_client import mongo_init
+from flask import Flask
+from flask_smorest import Api
+from flask_swagger_ui import get_swaggerui_blueprint
+
+ADDON_MARKETPLACE_PORT = os.environ.get("ADDON_MARKETPLACE_PORT") or 11102
+
+app = Flask(__name__)
+
+app.config["OPENAPI_VERSION"] = "3.0.2"
+app.config["API_TITLE"] = "Addon Marketplace Api"
+app.config["API_VERSION"] = "v1"
+app.config["OPENAPI_URL_PREFIX"] = "/docs"
+
+mongo_init(app)
+api = Api(app)
+
+
+# Register blueprints
+SWAGGER_URL = "/api/docs"
+API_URL = "/docs/openapi.json"
+swaggerui_blueprint = get_swaggerui_blueprint(
+    SWAGGER_URL,
+    API_URL,
+    config={"app_name": "Addon Marketplace"},
+)
+app.register_blueprint(swaggerui_blueprint)
+api.register_blueprint(marketplaceblp)
+
+
+@app.route("/", methods=["GET"])
+def health():
+    return "ok"
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=ADDON_MARKETPLACE_PORT, debug=False)
```

Comment:
Recommendation:

I am new to flask. During my own development I also stumbled upon this annoying warning:
![image](https://github.com/oakestra/oakestra/assets/65814168/2f6402fb-4300-4205-a689-24f543c048e4)

I think to overcome this is pretty easy to do.

Have a look here how to use a "production ready WSGI server instead":
dependency needed: waitress = "3.0.0"

I use slightly different dependencies but the core thing can be resolved the same:

```python
from http import HTTPStatus

import flask_openapi3
import flops_manager.api.blueprints as blps
from flops_manager.api.utils.consts import FLOPS_MANAGER_PORT
from waitress import serve

_info = flask_openapi3.Info(title="FLOps Manager API", version="1.0.0")
app = flask_openapi3.OpenAPI(__name__, info=_info)


@app.route("/", methods=["GET"])
def health():
    return {"message": "ok"}, HTTPStatus.OK


def handle_api():
    for blp in blps.blueprints:
        app.register_api(blp)

    # Note (AFAIK): 'waitress' comes with its own logger.
    # We are already using the flops logger.
    # Thus we have duplicated logger outputs (not just for API things but generally).
    # To avoid this the waitress ones gets muted.
    # If something should be logged that concerns the API it will be logged via the flops logger.
    serve(app, host="0.0.0.0", port=FLOPS_MANAGER_PORT, _quiet=True)
```